saddle-core/src/main/scala/org/saddle/scalar/ScalarTagDouble.scala

```diff
@@ -25,7 +25,7 @@ import org.saddle.array.Sorter
 object ScalarTagDouble extends ScalarTag[Double] {
   @inline def missing: Double = Double.NaN
   @inline def isMissing(v: Double): Boolean = (v != v)
-  @inline def notMissing(v: Double): Boolean = (v == v)
+  @inline override def notMissing(v: Double): Boolean = (v == v)
```

Comment:
is it possible to omit this overrie and rely on !isMissing() in the base class?

Reply:
sure, will do

Reply:
There overrides cannot be removed without introducing a binary incompatibility.